nxrm-aws-ha-helm/Chart.yaml

```diff
@@ -15,13 +15,13 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 43.0.0
+version: 45.0.0
```

Comment:
Should be 45.1.0 - We start the chart version with the last two sections of the appVersion